k8s/pkg/apis/longhorn/v1beta1/volume.go

```diff
@@ -101,6 +101,14 @@ const (
 	VolumeConditionReasonTooManySnapshots              = "TooManySnapshots"
 )
 
+type RestoreVolumeRecurringJobOpt string
```

Comment:
Why do we want these in v1beta1?

Reply:
Do you mean it could be just "string" or we don't need to handle `restoreRecurringJob` for v1beta1 API?

Reply:
I mean, why do we need make an update in `k8s/pkg/apis/longhorn/v1beta1/`?

Reply:
You are right, we do not need to update `k8s/pkg/apis/longhorn/v1beta1/`.
Removed. Much appreciated.